README.md

```
@@ -1,297 +1,33 @@
-    },
-    origin: '59.7.14.69',
-    url: 'https://httpbin.org/get'
-  }
-}
-```
+## 📍notice
+- 인기 사이트 조회 기능 추가
+
+## 테스트
+- database/schema.sql 226행~ 실행
+    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)